src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -359,14 +358,19 @@ public function initialized($id)
     public function getServiceIds()
     {
         $ids = array();
+        $reversedMethodMap = array_change_key_case(array_flip($this->methodMap), \CASE_LOWER);
         foreach (get_class_methods($this) as $method) {
             if (preg_match('/^get(.+)Service$/', $method, $match)) {
```

Comment:
Depends how this method _should_ work before/after compilation.

We could lazily load the method map when needed, which also happens when dumping. Im assuming the methods wont change during runtime anyway :)